src/VariableBorrowInterestRateStrategy.sol

```diff
@@ -155,7 +155,7 @@ contract VariableBorrowInterestRateStrategy is IDefaultInterestRateStrategy {
 
         // 1. Calculate the total debt and set the current variable borrow rate to the base rate.
 
-        vars.totalDebt = params.totalStableDebt + params.totalVariableDebt;
+        vars.totalDebt = params.totalVariableDebt;
```

Comment:
Lets just use `params.totalVariableDebt` everywhere for this, can remove `totalDebt` from the struct as well

Reply:
done